frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -49,7 +49,10 @@ export const dashboardLogic = kea({
                 loadDashboardItems: async ({ refresh = undefined } = {}) => {
                     try {
                         const dashboard = await api.get(
-                            `api/dashboard/${props.id}/?${toParams({ share_token: props.shareToken, refresh })}`
+                            `api/dashboard/${props.id ? `${props.id}/` : ''}?${toParams({
+                                share_token: props.shareToken,
+                                refresh,
+                            })}`
```

Comment:
This is off in a different way:
When an ID is passed in, we fetch the dashboard with that ID, which is great – but in the no ID case, we fetch the project's _list of dashboards_, which doesn't make sense. The core issue is that this logic doesn't fully understand that these "special" Home and History dashboards are not `posthog_dashboard` rows.

Reply:
I think this kind of stems from dashboardLogic not being used in the way it was meant to be used. DashboardLogic is being mounted whenever a dashboard item from a specific dashboard is loaded for the first time, but there are times when we want to show a dashboard item NOT in the context of a specific dashboard. Maybe this requires a different approach, like moving the refresh dashboard logic into dashboardItemLogic.

Apologies for the back and forth, this looks trickier than I initially thought.

Reply:
I'm going to revert this back to how we had it before, since there's no longer a scenario where props.id will be empty.